fuzz/generate-files.sh

```diff
@@ -11,7 +11,8 @@ source "$REPO_DIR/fuzz/fuzz-util.sh"
 cat > "$REPO_DIR/fuzz/Cargo.toml" <<EOF
 [package]
 name = "hex-fuzz"
-edition = "2018"
+edition = "2021"
+rust-version = "1.63.0"
```

Comment:
Since this is generated we should pull the value from single source.

Reply:
Unrelated to this PR, right? Can we just put this in, I spent way too long on this extremely annoying PR already.

Reply:
It's literally just replacing the line with `$(grep '^rust-version ' "$REPO_DIR/Cargo.toml")`